src/app/main-overview/components/ControlPlaneDetails.vue

```diff
@@ -13,7 +13,8 @@
           style="--columns: 4;"
         >
           <ResourceStatus
-            :total="store.getters['config/getMulticlusterStatus'] ? props.zoneOverviews.length : 1"
+            :total="store.getters['config/getMulticlusterStatus'] ? props.globalInsights.zones.cps.total : 1"
+            :online="store.getters['config/getMulticlusterStatus'] ? props.globalInsights.zones.cps.online : 1"
             data-testid="zone-control-planes-status"
           >
```

Comment:
This shows zones when in standalone:
<img width="1471" alt="image" src="https://github.com/kumahq/kuma-gui/assets/702355/0a2c5aea-ecc2-4fe2-9a19-03a8e62a3686">


Can we just remove the zones entry altogether by doing `<ResourceStatus v-if="store.getters['config/getMulticlusterStatus']" ... ` ?

Also could we make the mesh list take a the page in standalone?

Reply:
> Can we just remove the zones entry altogether by doing `<ResourceStatus v-if="store.getters['config/getMulticlusterStatus']" ... ` ?

I’ve changed this.

> Also could we make the mesh list take a the page in standalone?

Fixed.